segmenters/nlp/NLTKSentencizer/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jina
```

Comment:
please remove `jina` as dependency, since the Docker image is built with jina images as base